### PR TITLE
Test scheduler-pull

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/COM-1160

### CODE REVIEW

File: test/pull-scheduler/expected-stdout [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Plan for "pull" operation:
+  +  C main/extractor/ex-generic-v2/empty/schedules/scheduler
```
Comment (webrouse) [inline test/pull-scheduler/expected-stdout:2]: Pridany test pre `pull` scheduler konfiguracie.

File: test/pull-scheduler/in/.keboola/manifest.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "__all__"
+  ],
+  "ignoredComponents": [],
+  "branches": [
+    {
+      "id": %%TEST_BRANCH_MAIN_ID%%,
+      "path": "main"
+    }
+  ],
+  "configurations": [
```
Comment (webrouse) [inline test/pull-scheduler/in/.keboola/manifest.json:28]: V `in` mame iba 1 config.

File: test/pull-scheduler/out/.keboola/manifest.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  "configurations": [
+    {
+      "branchId": %%TEST_BRANCH_MAIN_ID%%,
+      "componentId": "ex-generic-v2",
+      "id": "%%TEST_BRANCH_MAIN_CONFIG_EMPTY_ID%%",
+      "path": "extractor/ex-generic-v2/empty",
+      "rows": []
+    },
+    {
+      "branchId": %%TEST_BRANCH_MAIN_ID%%,
```
Comment (webrouse) [inline test/pull-scheduler/out/.keboola/manifest.json:37]: V `out` pribudne scheduler config.

File: test/pull-scheduler/out/.keboola/manifest.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      "componentId": "keboola.scheduler",
+      "id": "%%TEST_BRANCH_MAIN_CONFIG_SCHEDULER_ID%%",
+      "path": "schedules/scheduler",
+      "relations": [
+        {
+          "componentId": "ex-generic-v2",
+          "id": "%%TEST_BRANCH_MAIN_CONFIG_EMPTY_ID%%",
```
Comment (webrouse) [inline test/pull-scheduler/out/.keboola/manifest.json:44]: So spravnou relaciou.

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**